src/pages/MyAccount.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Outlet } from "react-router-dom";
```

Comment:
There are two imports here from react-router-dom. These could be condensed into one import:
import { Link, Outlet } from "react-router-dom";

Reply:
Just saw the message. The issue has been fixed.